src/web/src/components/Login.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <b-form @submit="onSubmit" @reset="onReset" v-if="showForm">
+    <b-form-group id="input-group-1" label="Email address:" label-for="input-1">
+      <b-form-input
+        id="input-1"
+        v-model="form.email"
+        type="email"
+        required
+        placeholder="Enter email"
+      ></b-form-input>
+    </b-form-group>
+
+    <b-form-group id="input-group-2" label="Password:" label-for="input-2">
+      <b-form-input
+        id="input-2"
+        type="password"
+        v-model="form.password"
+        required
+        placeholder="Enter password"
+      ></b-form-input>
+    </b-form-group>
+
+    <b-button type="submit" variant="primary">Submit</b-button>
+  </b-form>
+</template>
+
+<script>
+import { login } from "@/services/UserService";
+
+export default {
+  name: "Login",
+  data() {
+    return {
+      form: {
+        email: "",
+        password: "",
+      },
+      showForm: true,
+    };
+  },
+  methods: {
+    onSubmit(evt) {
+      evt.preventDefault();
+      let userInfo = this.form;
+      console.log(userInfo);
+
+      login(userInfo)
+        .then((response) => {
+          console.log(response);
+          this.$cookies.set("sessionID", response.data.content["sessionID"]);
+          this.$cookies.set("userName", response.data.content["userName"]);
+          this.$cookies.set("userID", response.data.content["uid"]);
+          location.reload();
+        })
+        .catch((error) => {
+          console.log(error.response);
+          this.$bvToast.toast(
+            `Login Unsuccesful. Please double check your email and password, then try again!`,
+            {
+              title: "Invalid login",
+              variant: "danger",
+              noAutoHide: true,
+            }
+          );
+        });
+      this.$emit("submit");
+      // this.$refs["login-modal"].hide();
+    },
+    onReset(evt) {
+      evt.preventDefault();
+      // Reset our form values
+      this.form.email = "aaa1@wa.com";
+      this.form.password = "123456";
```

Comment:
Lets set these to empty strings, http://wa.com/ isn't gonna like it tho :)

Reply:
lool yuze be like "wa"